pkg/types/azure/defaults/platform.go

```diff
@@ -4,12 +4,70 @@ import (
 	"github.com/openshift/installer/pkg/types/azure"
 )
 
+var (
+	// This is likely not necessary since Dsv3-Series is generally available in all regions
+	defaultMachineClass = map[string]string{
```

Comment:
So if you look at [`defaultMachineClass`](https://github.com/openshift/installer/blob/0a743f45cedcb098523888dc9f6429de4c0ce7d3/pkg/types/aws/defaults/platform.go#L8-L14) it is a map of **overrides** for the region, and this list should be the same.

we pick a default, and use this list for overriding our general default.

Reply:
I removed the overrides. It is now empty.